### PR TITLE
showing how the devops works and some of the git commands

### PR DESCRIPTION
**Pull request by ibilalkayy**
Ahoy matey!

Thank ye for considering to contribute to our book project. Before ye submit your pull request, please make sure to check the following:

- [x] Your changes follow the book's style and conventions
- [ ] Your changes have been thoroughly reviewed and do not introduce any errors
- [ ] You have added appropriate documentation for your changes (if necessary)

#### Description of changes:

Showing how the DevOps works and adding some of the Git Commands

#### Issue(s) closed:

[List the issue(s) that this pull request resolves, if applicable]

Thanks for your contribution! We'll review your changes as soon as we can.

Yarrrr!

### CODE REVIEW

**Comment by ibilalkayy** [comment]
I added the data in a normal format but you can change it into a pirate language.

File: index.md [hunk 1]
```diff
@@ -841,12 +842,18 @@ Under DevOps, there be no more "silos". Sometimes, the two teams be mergin' into
...
+In some DevOps models, quality assurance and security teams may also become more tightly integrated with development and operations and throughout the application lifecycle. When security is the focus of everyone on a DevOps team and even the security teams be gettin' in on the action, with a thing called "DevSecOps". It's like havin' a lookout on the masthead, keepin' a keen eye out for danger at all times!
 
 The DevOps journey be like a never-endin' loop, always improvin' and changin', like the wind direction on a voyage. It's made up of eight phases, representin' the processes, capabilities, and tools needed for development and operations. And throughout it all, the teams be workin' and communicatin' together, like a good crew on a ship!
 
 So hoist the sails and set course for DevOps, me hearties! Yarrr!
 
+### **How DevOps Works?**
+
+Under the DevOps model, development and operations teams are no longer siled. Sometimes, these two teams are merged into a single team where the engineers work across the entire application lifecycle, from development and test to deployment to operations, and develop a range of skills not limited to a single function.
```
Comment (loftwah) [inline index.md:1096]: Is this meant to be siloed rather than siled? That is the only issue I can pick up so far.

Comment (ibilalkayy) [reply index.md:1096]: Sorry, it's my mistake. It is siled.

Comment (loftwah) [reply index.md:1096]: Cool just wanted to check so I can get a demo ready and show you how I translate it. Don't want to mess up any of your writing in the process if possible.

Comment (ibilalkayy) [reply index.md:1096]: No problem. If you have any questions. I am happy to answer them.

**Pull request closed without merging**